tests/test_tutorial/test_arguments/test_help/test_tutorial006.py

```diff
@@ -32,5 +33,6 @@ def test_script():
         [sys.executable, "-m", "coverage", "run", mod.__file__, "--help"],
         capture_output=True,
         encoding="utf-8",
+        env={**os.environ, "PYTHONIOENCODING": "utf-8"},
```

Comment:
Note: as in #824, not entirely clear to me why we need this, but the Windows tests fail if we don't have it.